src/NLog/MessageTemplates/ValueSerializer.cs

```diff
@@ -59,6 +59,8 @@ private ValueSerializer()
         private const int MaxValueLength = 512 * 1024;
         private const string LiteralFormatSymbol = "l";
 
+        private readonly MruCache<Enum, string> _enumCache = new MruCache<Enum, string>(10000);
```

Comment:
isn't that a bit big? (10000)

Reply:
No idea. Just chose the same as for the DefaultJsonSerializer. Maybe there are more than 10000 enums in the world?

Reply:
Should we change it to 5000 for the both of them ?

Reply:
was more thinking of 1000. Or even lower. Could not image your using 1000 different enum values in 1 project.

Reply:
Now changed it to 1500 (also for the Json-Converter)